geomesa-accumulo/geomesa-accumulo-tools/src/test/scala/org/locationtech/geomesa/accumulo/tools/ingest/IngestCommandTest.scala

```diff
@@ -16,20 +16,16 @@ import org.locationtech.geomesa.accumulo.MiniCluster
 import org.locationtech.geomesa.accumulo.tools.{AccumuloDataStoreCommand, AccumuloRunner}
 import org.locationtech.geomesa.utils.collection.SelfClosingIterator
 import org.locationtech.geomesa.utils.io.WithClose
-import org.locationtech.geomesa.utils.io.fs.LocalDelegate
+import org.locationtech.geomesa.utils.io.fs.LocalDelegate.StdInHandle
 import org.specs2.mutable.Specification
 import org.specs2.runner.JUnitRunner
 
-import java.io.{BufferedInputStream, ByteArrayInputStream, File, IOException, InputStream}
+import java.io.{BufferedInputStream, ByteArrayInputStream, File}
 import java.util.concurrent.atomic.AtomicInteger
-import scala.util.Try
 
 @RunWith(classOf[JUnitRunner])
 class IngestCommandTest extends Specification {
 
-  // This prevents different values of System.in across different tests from interfering with each other
-  sequential
-
```

Comment:
Is there a reason why we don't want the unit tests running sequentially?

My experience is that running them in parallel causes the `stdin` ingest tests to fail randomly and unpredictably, whereas running them sequentially will produce stable results.